db/database.js

```diff
@@ -1,16 +1,26 @@
 const Datastore = require("nedb");
 const db = new Datastore({ filename: "./db/user.db", autoload: true });
 
-const insert = function (data) {
-  db.insert(data, function (err, newData) {
-    console.log("inserted : ", newData);
+const insert = async function (data) {
+  const result = await new Promise((resolve, reject) => {
+    db.insert(data, function (err, newData) {
+      if (err) reject(err);
+      resolve(newData);
+    });
   });
+
+  return result;
 };
 
-const find = function (query) {
-  db.find(query, function (err, target) {
-    console.log("found : ", target);
+const find = async function (target) {
+  const result = await new Promise((resolve, reject) => {
+    db.find({ target }, function (err, data) {
```

Comment:
```suggestion
const find = async function (query) {
  const result = await new Promise((resolve, reject) => {
    db.find(query, function (err, data) {
```

`find` 함수의 파라미터로 받은 `target`이 실제 `db.find`에 `{ target }` 형태로 들어가는 둘다 그냥 `query`로 바꿔야 할 것으로 보입니다.

Reply:
앗...아 손가락이 미쳤나봅니다!